modules/oxide-components/src/main/ts/keynav/keyboard/NavigationTypes.ts

```diff
@@ -0,0 +1,11 @@
+import * as FlowType from './FlowType';
+import * as KeyingType from './KeyingType';
+import * as SpecialType from './SpecialType';
+import * as TabbingType from './TabbingType';
```

Comment:
This is one of the areas where pieces of Alloy were left out that I believe we should keep. Not in this PR, but `execution` in particular will remove the need for special-case handling of things like enter in an input component. `cyclic` is also useful for many UI paradigms. They all have their purpose.
https://github.com/tinymce/tinymce/blob/5dfffd8f62ec7cebc5b4b4c64e7aa3d755995b65/modules/alloy/src/main/ts/ephox/alloy/behaviour/keyboard/KeyboardBranches.ts#L10-L17

Reply:
We definitely need a follow-up ticket or leave a comment to make sure we will port them over in the future.

Reply:
Ticket created [here](https://ephocks.atlassian.net/browse/TINY-12533)